include/sajson.h

```diff
@@ -310,10 +310,10 @@ namespace sajson {
         }
 
         // valid iff get_type() is TYPE_OBJECT
-        string get_object_key(size_t index) const {
+        const char* get_object_key(size_t index) const {
```

Comment:
I think this is a regression because it no longer includes the length of the string, right?  Thus keys with embedded nulls would be too short.